performance-evaluation/util.sh

```diff
@@ -0,0 +1,156 @@
+function cleanLogs {
+    rm -f $SCHEDULER_HOME/logs/*.log
+}
+
+function waitServer {
+    echo "Wait until server Scheduler has been started"
+    lines="0"
+    while [ $lines -eq "0" ]
+    do
+        lines=$(grep "Get started" $SCHEDULER_LOG_FILE 2>/dev/null | wc -l )
+#		echo $lines
+            sleep 1
+    done
+
+}
+
+function startServer {
+    echo "Start scheduler with 0 nodes"
+    $SCHEDULER_HOME/bin/proactive-server -ln 0 &
+}
+
+
+
+function waitNode {
+    echo "Wait until node has been started"
+    lns="0"
+    while [ $lns -eq "0" ]
+    do
+        lns=$(grep "Connected to the resource manager at " $NODE_LOG_FILE 2>/dev/null | wc -l )
+#		echo $lns
+            sleep 1
+    done
+
+}
+
+function startNode {
+    echo Start node with N workers
+    $SCHEDULER_HOME/bin/proactive-node -w $N_WORKERS -Dproactive.node.reconnection.attempts=100 &
```

Comment:
We should specify here more parameters to start the node in order to set the communication protocol + the IP/port, e.g. with `-Dproactive.communication.protocol=pnp -r pnp://192.168.1.126:64738`.